src/Storage/Migration/Output/JsonFile.php

```diff
@@ -47,7 +48,7 @@ public function addRecord(array $data, $last)
         }
 
         // Generate the JSON string
-        $json = json_encode($data, JSON_PRETTY_PRINT);
+        $json = Json::dump($data, JSON_PRETTY_PRINT);
```

Comment:
This needs work. There's a false check below that needs to be converted to try/catch. Still don't know why you don't want escaped slashes and unicode here.